Adding migration to sidebar and removing old items (#5674)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/migration.md

## Pre-migration checklist

> Prior to your migration date, your dbt Cloud account admin will need to make some changes to your account. Most of your configurations will be migrated automatically, but a few of them will require manual intervention. 

Author: ```suggestion
Prior to your migration date, your dbt Cloud account admin will need to make some changes to your account. Most of your configurations will be migrated automatically, but a few will require manual intervention. 
```

If your account is scheduled for migration, you will see a banner indicating your migration date when you log in. If you don't see a banner, you don't need to take any action.

1. **IP addresses** &mdash; dbt Cloud will be using new IPs to access your warehouse after the migration. Make sure to allow inbound traffic from these IPs in your firewall and include it in any database grants. All six of the IPs below should be added to allowlists.
    * Old IPs: `52.45.144.63`, `54.81.134.249`, `52.22.161.231`
    * New IPs: `52.3.77.232`, `3.214.191.130`, `34.233.79.135`

> 2. **User invitations** &mdash; Any pending user invitations will be invalidated during the migration. You can resend the invitations once the migration is complete.

Reviewer: ```suggestion
2. **User invitations** &mdash; Any pending user invitations will be invalidated during the migration. You can resend the invitations after the migration is complete.
```

> 3. **SSO integrations** &mdash; If you've completed the Auth0 migration, your account SSO configurations will be automatically transferred. If you have not completed the Auth0 migration, we recommend completing that before the mult-cell migration to avoid service disruptions.

Reviewer: ```suggestion
3. **SSO integrations** &mdash; If you've completed the Auth0 migration, your account SSO configurations will be automatically transferred. If you haven't completed the Auth0 migration, dbt Labs recommends doing that before starting the mult-cell migration to avoid service disruptions.
```

Author: Keeping this as-is as they can't start the migration at this time

> 3. **SSO integrations** &mdash; If you've completed the Auth0 migration, your account SSO configurations will be automatically transferred. If you have not completed the Auth0 migration, we recommend completing that prior to the mult-cell migration to avoid service disruptions.

Author: ```suggestion
3. **SSO integrations** &mdash; If you've completed the Auth0 migration, your account SSO configurations will be automatically transferred. If you have not completed the Auth0 migration, we recommend completing that before the mult-cell migration to avoid service disruptions.
```
